src/sass/_shame.scss

```diff
@@ -5,11 +5,12 @@
 
 // message while loading react app
 .loading-message {
-  text-align: center;
   padding: 2em 0;
+  text-align: center;
 }
 
-body .row {
+
+.row {
```

Comment:
@cehsu I think we should remove any changes that affect the specificity of styles, since those could result in visual changes. So changes like removing qualifying elements. The other ones are fine.